### ISSUE TITLE
[BUG]: Insufficient color contrast ratio according to WCAG 2 AA

### ISSUE THREAD

**Issue by isaozler** [Trendyol/baklava#630]
### Issue description

Interactive components using the brand color palettes have Insufficient contrast ratios. I suggest you reconsider the color applications.

### Operating system

All

### Priority this issue should have

Medium (should be fixed soon)

### Please review the checkboxes that are applicable.

- [ ] This issue is directly related to React

**Comment by muratcorlu** [comment]
Hi @isaozler, thanks for reporting!

This is a known issue and a challenge since we use this "default" theme inside Trendyol and a change needs some considerations inside. But we plan to have a new theme with a new color palette, which will eventually be the default one for the open source version and will fulfill a11y requirements well. Stay tuned! 😊

**Comment by isaozler** [comment]
Indeed, most of the time, these decisions are not technical ones.

But sounds great 👍 excellent work, by the way. Keep It up 💪
